File: leetcode/Easy/2824. Count Pairs Whose Sum is Less than Target/solution.py
class Solution:
    def countPairs(self, nums: List[int], target: int) -> int:
        res = 0

        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                ans = nums[i] + nums[j]
                if ans < target:
                    res += 1

        return res

File: leetcode/Easy/2824. Count Pairs Whose Sum is Less than Target/solution.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var countPairs = function (nums, target) {
  let res = 0;

  for (let i = 0; i < nums.length; i++) {
    for (let j = i + 1; j < nums.length; j++) {
      let ans = nums[i] + nums[j];
      ans < target ? (res += 1) : false;
    }
  }

  return res;
};
